docs/swap-automation.md

```diff
@@ -0,0 +1,150 @@
+A mean-reversion strategy expects the prices to return to “normal” levels or a certain moving average following a temporary price spike. We can construct a similar strategy using the Liquidity Pool, where we will set upper and lower bound prices that will trigger a sell and a buy transaction respectively. If the behavior of the LP prices works as expected always returning to a certain moving average, we could profit by selling high and buying low. We will do this by swapping atestfet and CW20 with the Liquidity Pool, we refer to a sell transaction when we sell atestfet and get CW20 tokens, a buy transaction would be exactly the oposite.
+
+The code will require the following imports:
+
+```python
+from time import sleep
+from cosmpy.aerial.client import LedgerClient, NetworkConfig
+from cosmpy.aerial.contract import LedgerContract
+from cosmpy.aerial.faucet import FaucetApi
+from cosmpy.aerial.wallet import LocalWallet
+```
+
+We will define the *swap_native_for_cw20* function that trades `swap_amount` of atestfet from `wallet` for CW20 tokens by executing a `pair_contract`:
+
+
+```python
+def swap_native_for_cw20(swap_amount, pair_contract, wallet):
+    tx = pair_contract.execute({
+      "swap": {
+        "offer_asset": {
+          "info": {
+            "native_token": {
+              "denom": "atestfet"
+            }
+          },
+          "amount": str(swap_amount)
+        }
+      }
+    },sender= wallet, funds= str(swap_amount) + "atestfet")
+    print("swapping native for cw20 tokens")
+    tx.wait_to_complete()
+```
+Now, we will define the *swap_cw20_for_native* function that does exactly the opposite of the function defined above: trades `swap_amount` of CW20 tokens from `wallet` for atestfet. This time the CW20 `token_contract` is executed using the `pair_contract_address` and  a base64 encoded msg *{"swap":{}} = eyJzd2FwIjp7fX0*
+
+```python
+def swap_cw20_for_native(swap_amount, pair_contract_address, token_contract, wallet):
+    tx = token_contract.execute({
+      "send": {
+        "contract": pair_contract_address,
+        "amount": str(swap_amount),
+        "msg": "eyJzd2FwIjp7fX0="
+      }
+    },wallet)
+    print("swapping cw20 for native tokens")
+    tx.wait_to_complete()
+```
+Set the network configuration, define a local wallet and add some tokens to it using the FaucetApi
+
+```python
+# Define any wallet
+wallet =  LocalWallet.generate()
+
+# Network configuration
+ledger = LedgerClient(NetworkConfig.latest_stable_testnet())
+
+# Add tokens to wallet
+faucet_api = FaucetApi(NetworkConfig.latest_stable_testnet())
```

Comment:
Enclose in while loop the condition being the account has the amount you are looking for.